java/src/jmri/jmrit/roster/UpdateDecoderDefinitionAction.java

```diff
@@ -51,16 +51,21 @@ public void actionPerformed(ActionEvent e) {
                 if (decoder.getReplacementFamily() != null || decoder.getReplacementModel() != null) {
                     log.info("   Recommended replacement is family \"" + decoder.getReplacementFamily() + "\" model \"" + decoder.getReplacementModel() + "\"");
                 }
-                replacementFamily = (decoder.getReplacementFamily() != null) ? decoder.getReplacementFamily() : replacementFamily;
-                replacementModel = (decoder.getReplacementModel() != null) ? decoder.getReplacementModel() : replacementModel;
+                replacementFamily = (decoder.getReplacementFamily() != null) ? decoder.getReplacementFamily() : null;
```

Comment:
Thanks, agreed. It did cross my mind that it was redundant once I changed it, but forgot to clean it up. I'll do so.

Maybe if I'd run FindBugs it may have picked them up...